R/pkg/R/mllib.R

```diff
@@ -1852,9 +1867,9 @@ summary.treeEnsemble <- function(model) {
 
 #  Get the summary of a Random Forest Regression Model
 
-#' @return \code{summary} returns a summary object of the fitted model, a list of components
-#'         including formula, number of features, list of features, feature importances, number of
-#'         trees, and tree weights
+#' @return \code{summary} returns summary information of the fitted model, which is a list.
+#'         The list of components includes \code{ans} (formula, number of features, list of features,
+#'         feature importances, number of trees, and tree weights).
```

Comment:
the two places returns `summary.treeEnsemble(object)`. What shall I put in the `\code{}`?